DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Mappers/GitHub/GitHubMapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using GitHubExtension.Statistics.WebApi.Queries.Interfaces;
+
+namespace GitHubExtension.Statistics.WebApi.Mappers.GitHub
+{
+    public static class GitHubMapper
+    {
+        #region fields
+        private static int countWeeksInMonth = 4;
```

Comment:
Why you are like extract local variables into scope of class? It's a Mapper - STATIC class.